src/updateChangelog.ts

```diff
@@ -279,7 +275,12 @@ export default async function updateChangelog(
       releaseDate,
       genesisHash,
       owner,
-      repo
+      repo,
+    })
+    .data("settings", {
+      listItemIndent: "one",
+      tightDefinitions: true,
+      bullet: "-",
```

Comment:
This is the actual fix, right? Can we please split the remaining formatting off into another PR?

Reply:
Done. I kept the formatted `CHANGELOG.md`s as they are required for tests.